.github/workflows/copier.yml

```diff
@@ -0,0 +1,25 @@
+name: Copier template sync
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * 1'
+
+jobs:
+  update:
+    name: Sync with copier template
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.X'
+      - run: pip install copier
+      - run: copier update --skip-answered --vcs-ref=HEAD --conflict=rej
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          title: '[CRON] Sync with copier template'
```

Comment:
The scipp bot isn't here but let's try something anyway. We can run the action through workflow dispatch.